src/masoniteorm/commands/MigrateRefreshCommand.py

```diff
@@ -33,11 +31,11 @@ def handle(self):
         if self.option("seed") == "null":
             self.call(
                 "seed:run",
-                f"None --directory {self.option('seed-directory')} --connection {self.option('connection', 'default')}",
```

Comment:
Is this correct? Shouldn't this default to the `default` connection?

Reply:
right now this would default to the None connection right?

Reply:
@josephmancuso 
unsure about this?
you altered it in commit 18de4662

Reply:
@josephmancuso 
seems to work fine from the command line for me 

Do you need to do some testing?
Or can we resolve this?